### ISSUE TITLE
go get results in malformed file path

### ISSUE THREAD

**Issue by amnk** [ori-edge/k8s_gateway#204]
I am not sure of this is my environment, but on a fairly fresh Mac M1 and in Linux VM I get same results:
```
❯ go get github.com/ori-edge/k8s_gateway
go: downloading github.com/ori-edge/k8s_gateway v0.3.3
go: github.com/ori-edge/k8s_gateway: create zip: ...: malformed file path "...": invalid path element "..."
```
vs 0.3.2:
```
❯ go get github.com/ori-edge/k8s_gateway@v0.3.2
go: downloading github.com/ori-edge/k8s_gateway v0.3.2
go: downloading github.com/nginxinc/kubernetes-ingress v1.12.4
go: downloading sigs.k8s.io/gateway-api v0.4.3
go: downloading golang.org/x/tools v0.6.0
go: downloading google.golang.org/appengine v1.6.7
go: downloading golang.org/x/mod v0.8.0
go: downloading github.com/Azure/go-autorest/autorest/adal v0.9.20
go: downloading go.uber.org/zap v1.19.0
go: upgraded github.com/Azure/go-autorest/autorest/adal v0.9.18 => v0.9.20
go: added github.com/nginxinc/kubernetes-ingress v1.12.4
go: added github.com/ori-edge/k8s_gateway v0.3.2
go: upgraded go.uber.org/zap v1.17.0 => v1.19.0
go: added sigs.k8s.io/gateway-api v0.4.3
```
Did anyone ever see that? Maybe even fixed that?

**Comment by networkop** [comment]
Seems like I've accidentally left file named `...` in that release
<img width="699" alt="image" src="https://user-images.githubusercontent.com/12479763/227705895-ca72c457-820d-4e20-979a-6f8676ce841c.png">

It's no longer there, so subsequent releases should be fine

**Comment by amnk** [comment]
@networkop thanks for fast reply! Maybe it is possible to cutoff fresh release to fix this?

**Comment by networkop** [comment]
fixed in 0.3.4
